src/components/Slider/index.tsx

```diff
@@ -43,18 +44,19 @@ export default function Slider(props: SliderProps): JSX.Element {
   };
 
   const [currentValue, setCurrentValue] = useState(defaultValue);
-  const handleChange = (event: React.SyntheticEvent | Event, value: number | number[]) => {
-    if (!Array.isArray(value)) {
-      if (value !== currentValue) {
-        setCurrentValue(value);
-        onChange(value);
+  const handleChange = (event: React.SyntheticEvent | Event, val: number | number[]) => {
+    if (!Array.isArray(val)) {
+      if (val !== currentValue) {
+        setCurrentValue(val);
```

Comment:
do we need the current value?

Reply:
Ah, i neglected that. I can probably get rid of it